.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "Feedback Forensics",
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.10-bullseye",
```

Comment:
Do we maybe want to go for 3.13? 3.10 is quite old, I am currently using 3.13. See here for version timelines: https://devguide.python.org/versions/

Reply:
This is modeled after the requirement in `pyproject.toml`, why not stick to it as long as we don't use any >3.10 features?

Reply:
Don't think we use anything now that requires >3.10 - so don't mind too much. We may want to add cross-python version tests via GH actions in the future to more explicitly test versions.